### DISCUSSION TITLE
django logo color changing

### DISCUSSION THREAD

**Discussion by mikeoliversinag** [fabiocaccamo/django-admin-interface#297] [Q&A] [answered]
It is really cool how the logo with the word django can be changed with a color in the theme editor.  I want the same thing but instead of  'django' I want to use my own word but keep the ability to change that word's color.

Can I do that?

**Comment by fabiocaccamo** [comment, marked as answer]
@mikeoliversinag it is not possible to change the django logo text because it is an `.svg` image. If you want to use your own text/color you can hide the logo, show the title and customize its text/color (you can do this in the theme admin).
